examples/lock-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -51,6 +52,18 @@ void DeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, intptr_
     case DeviceEventType::kSessionEstablished:
         OnSessionEstablished(event);
         break;
+
+    case DeviceEventType::kInterfaceIpAddressChanged:
```

Comment:
Would it be simpler or clearer to place all `chip::app::Mdns::StartServer();` calls in `DeviceEventCallback`?

Reply:
This is how it is done for the all-clusters-app example. So I am not sure that moving it in `DeviceEventCallback` would be clearer.

Reply:
Sure, OK.  I mean it's just moving the calls one function deeper.

No problem.